utility/utils.py

```diff
@@ -754,3 +755,124 @@ def download_openshift_installer(version=defaults.INSTALLER_VERSION):
         run_cmd(f"tar xzvf {tarball}")
 
     return installer_filename
+
+
+def generate_yaml_from_template(file_, **kwargs):
```

Comment:
I would keep all templating utils in this [templating module](https://github.com/red-hat-storage/ocs-ci/blob/master/utility/templating.py) or class itself if possible as static method for example.

For example we can define custom filters there which are part of env defined there like to_nice_yaml and so on.

Reply:
wasn't aware of this file.
Will move it there in case this is not a dup